ledger/src/blockstore_meta.rs

```diff
@@ -51,8 +51,6 @@ pub struct ShredIndex {
 pub struct ErasureMeta {
     /// Which erasure set in the slot this is
     pub set_index: u64,
-    /// First coding index in the FEC set
-    pub first_coding_index: u64,
```

Comment:
hmmm since these are stored in the database, is this going to cause compatibility issues if we try to fetch old ErasureMeta's?

Reply:
you are right. fixed